crates/router/src/core/errors/api_error_response.rs

```diff
@@ -18,129 +18,126 @@ pub enum ErrorType {
-    #[error(error_type = ErrorType::InvalidRequestError, code = "IR_04", message = "The HTTP method is not applicable for this API.")]
+    #[error(error_type = ErrorType::InvalidRequestError, code = "IR_03", message = "The HTTP method is not applicable for this API")]
     InvalidHttpMethod,
-    #[error(error_type = ErrorType::InvalidRequestError, code = "IR_05", message = "Missing required param: {field_name}.")]
+    #[error(error_type = ErrorType::InvalidRequestError, code = "IR_04", message = "Missing required param: {field_name}")]
     MissingRequiredField { field_name: String },
     #[error(
-        error_type = ErrorType::InvalidRequestError, code = "IR_06",
-        message = "{field_name} contains invalid data. Expected format is {expected_format}."
+        error_type = ErrorType::InvalidRequestError, code = "IR_05",
+        message = "{field_name} contains invalid data. Expected format is {expected_format}"
     )]
     InvalidDataFormat {
         field_name: String,
         expected_format: String,
     },
-    #[error(error_type = ErrorType::InvalidRequestError, code = "IR_07", message = "{message}")]
+    #[error(error_type = ErrorType::InvalidRequestError, code = "IR_06", message = "{message}")]
     InvalidRequestData { message: String },
-    /// Typically used when a field has invalid value, or deserialization of the value contained in
-    /// a field fails.
-    #[error(error_type = ErrorType::InvalidRequestError, code = "IR_07", message = "Invalid value provided: {field_name}.")]
+    /// Typically used when a field has invalid value, or deserialization of the value contained in a field fails.
+    #[error(error_type = ErrorType::InvalidRequestError, code = "IR_07", message = "Invalid value provided: {field_name}")]
     InvalidDataValue { field_name: &'static str },
-    #[error(error_type = ErrorType::InvalidRequestError, code = "IR_07", message = "Client secret was not provided")]
+    #[error(error_type = ErrorType::InvalidRequestError, code = "IR_08", message = "Client secret was not provided")]
     ClientSecretNotGiven,
-    #[error(error_type = ErrorType::InvalidRequestError, code = "IR_07", message = "The client_secret provided does not match the client_secret associated with the Payment.")]
+    #[error(error_type = ErrorType::InvalidRequestError, code = "IR_09", message = "The client_secret provided does not match the client_secret associated with the Payment")]
     ClientSecretInvalid,
-    #[error(error_type = ErrorType::InvalidRequestError, code = "IR_07", message = "Customer has existing mandate/subsciption.")]
+    #[error(error_type = ErrorType::InvalidRequestError, code = "IR_10", message = "Customer has active mandate/subsciption")]
     MandateActive,
-    #[error(error_type = ErrorType::InvalidRequestError, code = "IR_07", message = "Customer has already redacted.")]
+    #[error(error_type = ErrorType::InvalidRequestError, code = "IR_11", message = "Customer has already been redacted")]
```

Comment:
I'm seeing people using redacted customer. Is this some payment industry term? Redacted usually refers to only things hidden/omitted in documents?

Reply:
Redacted is used to mean that the customer information was removed on the customer's request to comply with GDPR rules.